docs/reference/reference-training/training/api-pytorch-reference.rst

```diff
@@ -8,105 +8,112 @@
 | :doc:`/training/apis-howto/api-pytorch-ug` |
 +--------------------------------------------+
 
-*************************************
- ``determined.pytorch.PyTorchTrial``
-*************************************
+***************
+ PyTorch Trial
+***************
+
+``determined.pytorch.PyTorchTrial``
+===================================
 
 .. autoclass:: determined.pytorch.PyTorchTrial
    :members:
    :inherited-members:
    :member-order: bysource
    :special-members: __init__
 
-********************************************
- ``determined.pytorch.PyTorchTrialContext``
-********************************************
+``determined.pytorch.PyTorchTrialContext``
+==========================================
 
 .. autoclass:: determined.pytorch.PyTorchTrialContext
    :members:
    :inherited-members:
```

Comment:
there are no longer inherited members on this object

Reply:
removed :inhertied-members: